### PR TITLE
Add xfixes feature

### PR DESCRIPTION
**Pull request by conqp**
- [*] Tested on an x11 machine
- [*] Compilation warnings were addressed
- [*] `cargo fmt` has been run on this branch
- [*] `cargo doc` builds successfully
- [*] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users
- [*] Updated documentation to reflect any user-facing changes
- [-] Created or updated an example program if it would help users understand this functionality

Fixes https://github.com/AltF02/x11-rs/pull/159

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  
 ### Changed
 
+## [2.20.1] - 2022-11-24
+### Added
+- xfixes feature
+
```
Comment (AltF02) [inline CHANGELOG.md:15]: Could you possibly move this into unreleased? I'll create a 2.20.1 release after merging, but I'd like to keep it a separate commit

**Review by AltF02 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (AltF02) [reply CHANGELOG.md:15]: Same with the 2.20.1 change in Cargo.toml 😄

Comment (conqp) [reply CHANGELOG.md]: Done.

**Review by AltF02 — APPROVED** [review]
Cheers!

**Comment by AltF02** [comment]
Released at https://crates.io/crates/x11/2.20.1

**Pull request merged**